app/models/user.rb

```diff
@@ -27,19 +33,29 @@
 #  verified                  :boolean
 #  created_at                :datetime         not null
 #  updated_at                :datetime         not null
+#  invited_by_id             :bigint
 #  zendesk_user_id           :bigint
 #
 # Indexes
 #
 #  index_users_on_email                 (email) UNIQUE
+#  index_users_on_invitation_token      (invitation_token) UNIQUE
+#  index_users_on_invitations_count     (invitations_count)
+#  index_users_on_invited_by_id         (invited_by_id)
 #  index_users_on_reset_password_token  (reset_password_token) UNIQUE
 #
+# Foreign Keys
+#
+#  fk_rails_...  (invited_by_id => users.id)
+#
 class User < ApplicationRecord
-  devise :database_authenticatable, :lockable, :validatable, :timeoutable, :trackable,
+  devise :database_authenticatable, :lockable, :validatable, :timeoutable, :trackable, :invitable,
          :omniauthable, omniauth_providers: [:zendesk]
 
   attr_encrypted :access_token, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
 
+  validates_presence_of :name
```

Comment:
I think it'd be good to validate there are no `User` records in prod/elsewhere with a blank `name`.